web/src/main/java/org/fao/geonet/proxy/URITemplateProxyServlet.java

```diff
@@ -116,6 +141,65 @@ protected void initTarget() throws ServletException {
                 doForwadHostPrefixPathString != null ? doForwadHostPrefixPathString : "";
         }
         super.initTarget();
+
+        // If baseUri is not set in web.xml try to resolve it from java properties or environment variables.
+        // The name is composed by the application base url,  servlet name, a point or an underscore and baseurl word.
+        if (StringUtils.isBlank(targetUriTemplate)) {
```

Comment:
Should be `targetUri` instead of `baseUri`

Reply:
Fixed